src/components/Experiment/Experiment.tsx

```diff
@@ -24,6 +24,9 @@ import {
  *
  * The code always defaults to version A if the experiment is not running
  * (or ended) and we still have both versions in the code.
+ *
+ * Note: The optimize script needs to be uncommented from index.html before
+ * starting experiments.
```

Comment:
FWIW I wonder if there's an npm module for google-optimize that we could dynamically import here.  That'd be ideal, since then we wouldn't load the script until you actually get to a page that has the experiment.  Something to maybe poke at briefly next time we do an experiment.